app/admin/rules/template.hbs

```diff
@@ -13,4 +13,5 @@
   </span>
 
   {{outlet}}
+  {{ember-notify messageStyle='bootstrap'}}
```

Comment:
Let's also increase the notification time.  It feels a little bit too quick and hard to catch the message.  Something like this?
```
{{ember-notify messageStyle='bootstrap' closeAfter=4000}}
```